hexrd/instrument.py

```diff
@@ -3729,13 +3964,13 @@ def __init__(self, image_series_dict, instrument, plane_data,
             plane_data, image_series_dict,
             active_hkls=active_hkls, threshold=threshold,
             tth_tol=None, eta_tol=eta_step)
-        
+
         # pack all detectors with masking
         # FIXME: add omega masking
         data_store = []
         for i_ring in range(n_rings):
-            # for convenience
-            map_shape = eta_mapping[det_key][i_ring].shape
+            # for convenience grab map shape from first
+            map_shape = next(iter(eta_mapping.values())).shape[1:]
```

Comment:
This can be outside of the loop if we just grab the first one, right?

Reply:
ah yes, you are correct!